kratos/processes/from_json_check_result_process.h

```diff
@@ -0,0 +1,921 @@
+//    |  /           |
+//    ' /   __| _` | __|  _ \   __|
+//    . \  |   (   | |   (   |\__ \.
+//   _|\_\_|  \__,_|\__|\___/ ____/
+//                   Multi-Physics
+//
+//  License:		 BSD License
+//					 Kratos default license: kratos/license.txt
+//
+//  Main authors:    Vicente Mataix Ferrandiz
+//
+
+#if !defined(KRATOS_FROM_JSON_CHECK_RESULT_PROCESS_H_INCLUDED )
+#define  KRATOS_FROM_JSON_CHECK_RESULT_PROCESS_H_INCLUDED
+
+// System includes
+
+// External includes
+#include <unordered_map>
+
+// Project includes
+#include "processes/process.h"
+#include "includes/model_part.h"
+#include "includes/table.h"
+#include "includes/kratos_parameters.h"
+
+namespace Kratos
+{
+
+///@name Kratos Globals
+///@{
+
+///@}
+///@name Type Definitions
+///@{
+
+    /// The definition of the index type
+    typedef std::size_t IndexType;
+
+    /// The definition of the sizetype
+    typedef std::size_t SizeType;
+
+///@}
+///@name  Enum's
+///@{
+
+///@}
+///@name  Functions
+///@{
+
+/**
+ * @brief Auxiliar method to obtain the corresponding modelpart from a given settings and Model
+ * @param rModel The model where the where the simulation is performed
+ * @param ThisParameters The parameters of configuration
+ */
+ModelPart& GetModelPartFromModelAndSettings(
+    Model& rModel,
+    Parameters ThisParameters
+    )
+{
+    const std::string& r_model_part_name = ThisParameters["model_part_name"].GetString();
+    const std::string sub_model_part_name = ThisParameters.Has("sub_model_part_name") ? ThisParameters["sub_model_part_name"].GetString() : "";
+
+    if (sub_model_part_name == "") {
+        return rModel.GetModelPart(r_model_part_name);
+    } else {
+        return rModel.GetModelPart(r_model_part_name).GetSubModelPart(sub_model_part_name);
+    }
+}
```

Comment:
Same comment above. If we want this to be a  general Kratos method I'd implement in another place, so we clearly now it is in there.